### DISCUSSION TITLE
Still have not get Student benefits 4 days after Applying

### DISCUSSION THREAD

**Discussion by 2N3ll** [community/community#133299] [GitHub Education]
### Hi everyone,

Hi Everyone, when I look at the status of the education pack it says “Your request was approved. [Your benefits will be available soon.]” however it’s about to hit 5 days now and not to shaw if any additional info is required as I have received no emails regarding it

**Comment by davevad93** [comment]
Hi @xyzkonik  ,

Don't worry, you don't have to submit any additional info, you just have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @xyzkonik was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470

**Comment by ghost** [comment]
Contact GitHub Support: If you haven't received any communication regarding your application status, reach out to GitHub support directly. You can visit the GitHub Support page or contact them via email to inquire about the status of your application.
